src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -96,6 +97,12 @@ def build_string_from_input(prompt, bos_token, image_seq_len, image_token, num_i
         image_token (`str`): The image token.
         num_images (`int`): Number of images in the prompt.
     """
+    if image_token in prompt:
+        warnings.warn(
+            f"The image token {image_token} is already present in the prompt. No need to manually add {image_token} in the prompt for this model."
+            f" Removing all {image_token} and adding ({image_token}) * image_seq_len * num_images at the start of the prompt."
+        )
+        prompt = prompt.replace(image_token, "")
```

Comment:
in one of prev PRs it was decided that we should support Paligemma in similar fashion as llava models, thus the user must provide "image" token, especially when doing multi-image generation. I don't think we should be raising warning here misleading users, since we also raise warning when no 'image' token is passed (lines 271-275 below)

Reply:
Oh I hadn't seen that, will remove it. Thanks